packages/next/client/on-demand-entries-client.js

```diff
@@ -4,7 +4,16 @@ import Router from 'next/router'
 import fetch from 'unfetch'
 
 const { hostname, protocol } = location
-const wsProtocol = protocol.includes('https') ? 'wss' : 'ws'
+const isHttps = protocol.includes('https')
+
+//
+// Cannot use `wss` until we can spin up ws server with ssl certs
+//
+const wsProtocol = 'ws'
```

Comment:
Won't this cause connection issues because of secure (https) vs insecure (ws) connection?

Reply:
I tested both Safari and Chrome allow `https` -> `ws://localhost`. If you try to use a hostname other than localhost it will be rejected, however.

Reply:
Just confirmed this works with Firefox also.